From c7fdff8eea6485373fb75f1402c6ce58bb0a1813 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Fri, 13 Mar 2015 10:02:29 +0100
Subject: main: Fix Videos icon not showing in volume control

Forcefully set PulseAudio icon property. Seems that PulseAudio's client
libraries aren't quite working things out in some cases.

https://bugzilla.gnome.org/show_bug.cgi?id=745874

diff --git a/src/totem.c b/src/totem.c
index 2600921..3152ae0 100644
--- a/src/totem.c
+++ b/src/totem.c
@@ -72,7 +72,7 @@ main (int argc, char **argv)
 	g_set_application_name (_("Videos"));
 	gtk_window_set_default_icon_name ("totem");
 	g_setenv("PULSE_PROP_media.role", "video", TRUE);
-
+	g_setenv("PULSE_PROP_application.icon_name", "totem", TRUE);
 
 	/* Build the main Totem object */
 	totem = g_object_new (TOTEM_TYPE_OBJECT,
-- 
cgit v0.10.2

